junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvParserFactory.java

```diff
@@ -27,15 +27,18 @@
 	private static final char SINGLE_QUOTE = '\'';
 	private static final char DOUBLE_QUOTE = '"';
 	private static final char EMPTY_CHAR = '\0';
+	private static final int DEFAULT_MAX_CHARS_PER_COLUMN = 4096;
```

Comment:
**Team Decision**: In line with related comments, this constant is no longer necessary.

Reply:
Removed.